ionic_user_interface/src/router/index.ts

```diff
@@ -56,6 +56,32 @@ const routes: Array<RouteRecordRaw> = [
       }
     },
   },
+  {
+    path: '/forgotPassword',
+    name: 'ForgotPassword',
+    component: () => import('@/views/ForgotPassword.vue'),
+    beforeEnter: (_, __, next) => {
+      if (localStorage.getItem('isLoggedIn') === 'yes') {
```

Comment:
Actually, can help try if it's possible to import the providers here? Instead of manipulating localStorage directly